src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -49,8 +65,9 @@ function logCapturedError(capturedError : CapturedError) : void {
 
     console.error(
       `${componentNameMessage} You should fix this error in your code. ${errorBoundaryMessage}\n\n` +
-      `${error.stack}\n\n` +
-      `The error was thrown in the following location: ${componentStack}`
+      `${errorSummary}\n\n` +
+      `The error is located at: ${componentStack}\n\n` +
+      `The error was thrown at: ${formattedCallStack}`
```

Comment:
I think you could also get away with just showing the call stack without any intro line before it.